jep/0000/README.adoc

```diff
@@ -0,0 +1,316 @@
+* centos7
+* latest
+* slim
+
+Change to JDK 11 in the link:https://hub.docker.com/r/jenkins/jenkins[long term support controller images] that do not include an explicit JDK in their tag, like:
+
+* 2.302.1
+* 2.302.1-alpine
+* 2.302.1-centos7
+* 2.302.1-slim
+* lts
+* lts-alpine
+* lts-centos7
+* lts-slim
+
+Change to JDK 11 in the link:https://hub.docker.com/r/jenkins/agent/[agent images] that do not include an explicit JDK in their tag, like:
+
+* 4.9-1
+* 4.9-1-alpine
+* 4.9-1-windowsservercore-ltsc2019
+* alpine
+* archlinux
+* latest
+* latest-archlinux
+* stretch - stop publishing updates to this image, replace with buster image
+
+Change to JDK 11 in the link:https://hub.docker.com/r/jenkins/inbound-agent[inbound agent images] that do not include an explicit JDK in their tag, like:
+
+* 4.9-1
+* 4.9-1-alpine
+* 4.9-1-windowsservercore-ltsc2019
+* alpine
+* latest
+
+Change to JDK 11 in the link:https://hub.docker.com/r/jenkins/ssh-agent/[outbound (ssh) agent images] that do not include an explicit JDK in their tag, like:
+
+* 2.1.0-alpine
+* 3.0.0
+* 3.0.0-alpine
+* alpine
+* latest
+* stretch - no change, retire the image
+
+=== Add new tags and images
+
+Additional tags will be added for those users who require a Java 8 image.
+
+**Provide additional Docker images for Java 8** in case users have critical requirements that must use Java 8.
+
+Add JDK 8 tags for the link:https://hub.docker.com/r/jenkins/jenkins[weekly controller images] as a fallback for users requiring JDK 8:
+
+* latest-jdk8
+* slim-jdk8
+* centos7-jdk8
+* alpine-jdk8
+
+Add JDK 8 tags for the link:https://hub.docker.com/r/jenkins/jenkins[long term support controller images] that do not include an explicit JDK in their tag, like:
+
+* lts-jdk8
+* lts-slim-jdk8
+* lts-centos7-jdk8
+* lts-alpine-jdk8
+
+Add JDK 8 tags to the link:https://hub.docker.com/r/jenkins/inbound-agent[inbound agent images] that do not include an explicit JDK in their tag, like:
+
+* 4.9-1-jdk8
+* 4.9-1-jdk8-alpine
+* 4.9-1-jdk8-windowsservercore-ltsc2019
+* alpine-jdk8
+* latest-jdk8
+
+Add JDK 8 tags to the link:https://hub.docker.com/r/jenkins/ssh-agent/[outbound (ssh) agent images] that do not include an explicit JDK in their tag, like:
+
+* 2.1.0-alpine-jdk8
+* 3.0.0-jdk8
+* 3.0.0-alpine-jdk8
+* alpine-jdk8
+* latest-jdk8
+* stretch no change, retire the image
+
+Some of the existing tags are for operating systems that have ended their standard support life.
+Add new operating system tags to use actively maintained operating systems.
+
+**Add Docker images for Debian 10 (buster)** in those images that include a link:https://www.debian.org/releases/stretch/[Debian 9 (stretch)] image.
+Debian 9 switched to link:https://wiki.debian.org/LTS[long term support] July 9, 2020.
+Debian 9 link:https://wiki.debian.org/LTS[long term support] will end June 30, 2022.
+
+Add Debian Buster image to the outbound (ssh) agent images to replace Debian stretch:
+
+* buster
+
+=== Stop publishing obsolete tags
```

Comment:
Shall we add the windows 18.09 images here too, e.g https://github.com/jenkinsci/docker-agent/blob/master/11/windows/windowsservercore-1809/Dockerfile

Reply:
Yes, very good idea.  I dropped them from the tag lists but did not add them to the "remove obsolete tags" section.  Will do that.

Reply:
@timja https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/base-image-lifecycle indicates that 1809 base images are supported until 2024.  Can you help me understand what I'm missing that leads us to want to drop the 1809 based Windows images?

Reply:
From what I can tell Windows server 18.09 is out of support except for ltsc which is a specific version (that we do use for the 2019 version, https://github.com/jenkinsci/docker/blob/master/11/windows/windowsservercore-2019/hotspot/Dockerfile#L2)

https://docs.microsoft.com/en-us/windows/release-health/status-windows-10-1809-and-windows-server-2019